contents/julia_basics.md

```diff
@@ -1838,7 +1839,7 @@ We share code, data, tables, figures and so on.
 Behind everything, there is the **operating system (OS) filesystem**.
 In a perfect world, the same program would give the **same** output when running on **different** operating systems.
 Unfortunately, that is not always the case.
-One instance of this is the difference between Windows paths, such as `C:\\user\john\`, and Linux paths, such as `/home/john`.
+One instance of this is the difference between Windows paths, such as `C:\user\john`, and Linux paths, such as `/home/john`.
```

Comment:
```suggestion
One instance of this is the difference between Windows paths, such as `C:\\user\john`, and Linux paths, such as `/home/john`.
```

Windows uses the double inverted slash bar thing before a drive.

Reply:
Sorry, but I was not sure if you just wanted to give a path as would normally be given/shown in the Windows explorer or `cmd`/`powershell`. There would be only *one* backslash each. When I do e.g.

```julia
root = dirname(@__FILE__)
```

in my Julia test file I get double backlashes *everywhere*, i.e. not only after the drive letter. So I e.g. get `d:\\my\\path\\to\\Julia\\test.jl`.

So when the intent was to show a valid Windows path in Julia the above instance should be changed to

```md
... such as `C:\\user\\john`, ...
```

Reply:
Those double backslashes aren't always shown :)

```julia
julia> raw"foo\bar"
"foo\\bar"

julia> raw"foo\bar" |> print
foo\bar
```

Reply:
@rikhuijzer, (as a reminder, I have just started with Julia) and I can see your argument. But can you image an output (of `dirname(@__FILE__) on Windows) where you will get a **mix** of single and double backslashes? Because that is what currently is written in the book ...

My suggestion was and is to show either **all** instances of backslashes as "singles" or as "doubles" (but not as a mixture of both).

Reply:
Fixed in https://github.com/JuliaDataScience/JuliaDataScience/pull/224.